ui/src/components/HelpAccordion.tsx

```diff
@@ -89,10 +89,9 @@ export function HelpAccordion({ title, body }: HelpDrawerProps): JSX.Element {
   const [isHover, setIsHover] = useState(false);
   return (
     <StyledAccordion
-      boxShadow={0}
       onMouseEnter={() => setIsHover(true)}
       onMouseLeave={() => setIsHover(false)}
-      isHover={isHover}
+      ishover={isHover ? 1 : 0}
```

Comment:
Changed b/c of console error: https://github.com/styled-components/styled-components/issues/1198